.travis.yml

```diff
@@ -27,6 +27,7 @@ before_install:
   - rvm use 2.1.5 --install
   - nvm install 0.12.2
   - nvm use 0.12.2
+  - ./transform.sh
```

Comment:
Sorry Maybe I didn't make myself clear in the initial comments.  Because this is touching a lot of different places in the code I thought it would be best to have the transformations be done by a script, that way if other stuff is merged in, we don't have to worry as much about merge conflicts.

If I get the needed +1s on the change I will run the script, remove it and any mention of it in the code and check in the result before merging it to master.

I added this line so that travis could test the fully transformed code.